snuba/utils/streams/batching.py

```diff
@@ -292,8 +292,10 @@ def _run_once(self) -> None:
         except self.__recoverable_errors:
             return
 
-        assert self.__processor is not None, "received message without active processor"
```

Comment:
I am not sure I get the failure mode here. was this assertion failing when we received empty messages before the partition was assigned? Or did the poll timeout expire before a partition being assigned ?

Reply:
> was this assertion failing when we received empty messages before the partition was assigned

Yes, this.